doc/src/create_atoms.rst

```diff
@@ -316,6 +371,12 @@ the atoms around the rotation axis is consistent with the right-hand
 rule: if your right-hand's thumb points along *R*, then your fingers
 wrap around the axis in the direction of rotation.
 
+The *radiusscale* keyword only applies to the *mesh* style and allows to
+adjust the radius of created particles, provided this is supported by
+the atom style.  Its value is a scaling factor (default: 1.0) that is
+applied to the radius inferred from the size of the individual triangles
```

Comment:
What is the "size" of the triangle?  Its longest or shortest edge length, or something else?

Reply:
Also, possibly I missed it, but I think advice should be given to turn-off particle interactions between pairs of wall particles, e.g. via neigh_modify exclude, since it is likely there will be overlaps, either for finite-size or point particles.

Reply:
How the radius is inferred (average of distance from center) is discussed with the bisect algorithm.